app/views/layouts/application.html.erb

```diff
@@ -39,6 +39,12 @@
       <%= render 'govuk_publishing_components/components/phase_banner', phase: @content_item.phase %>
     <% end %>
 
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render 'shared/save_page_confirmation' %>
```

Comment:
This partial contains logic for whether the contents are shown or hidden, but that doesn't wrap these containing grid divs, which means they'll always be rendered on the page regardless. Would it be better to move them inside the partial, inside the `if` statement?

Reply:
Very good point, have done as suggested